xml/http_servers.xml

```diff
@@ -3376,12 +3376,13 @@
       <param pos="1" name="service.version"/>
    </fingerprint>
 
-   <fingerprint pattern="^Boa/(\d+\.[a-zA-Z0-9\.\-]+)">
+   <fingerprint pattern="^Boa/([\d\.]+(?:[a-zA-Z0-9\.\-]*))">
```

Comment:
Don't need to escape the - because it's at the end (or if you put it at the start).
Don't need to escape the . because it's not a wild card inside a block.